eng/Versions.props

```diff
@@ -23,6 +23,9 @@
     <MicrosoftBclAsyncInterfacesVersion>1.0.0-preview7.19311.1</MicrosoftBclAsyncInterfacesVersion>
```

Comment:
You can delete MicrosoftBclAsyncInterfacesVersion and the subscription from Version.Details.xml

Reply:
thanks.